src/Relationship.php

```diff
@@ -11,73 +11,81 @@
 
 namespace Tobscure\JsonApi;
 
-class Relationship
+use JsonSerializable;
+
+class Relationship implements JsonSerializable
```

Comment:
how we support if relationship is empty? now we return {data: id: {null, type: type}} but it should be {data: null}

Reply:
What happens if you return `Relationship::fromData(null)`?